docs/README.md

```diff
@@ -2,11 +2,9 @@
 
 You can query a node and interact with the Polkadot network using Javascript. This guide describes the Polkadot Javascript API.
 
-{% include "SUMMARY.md" %}
```

Comment:
Removed since the "hidden" override doesn't apply here. And since it is in the sidebar via SUMMARY.md, we are not losing navigation